addon/types/model_export/model.py

```diff
@@ -7,6 +7,7 @@
 from pathlib import Path
 from traceback import print_exc
 from ... utils import common
+from ... utils import game as game_utils
```

Comment:
I don't love importing as; would it be possible to import utils itself and then use utils.common and utils.game everywhere in this file?
Don't pay it any mind if you think it's a bad idea.

Reply:
Added this as a lazy workaround for a parameter with the name `game` :trollface: 

will change though